phpstan.neon

```diff
@@ -4,6 +4,13 @@ parameters:
 		- src
 		- tests
 		- bdi
+	ignoreErrors:
+		-
+			message: '#Method .* return type has no value type specified in iterable type .*.#'
+			path: tests/Browser/GoogleChrome/VersionResolverTest.php
+		-
+			message: '#.* no value type specified in iterable type .*.#'
+			path: src/Browser/GoogleChrome/VersionResolver.php
```

Comment:
You can fix these errors by adding a docblock with the iterable type.

For example:
```php
/**
  * @return array<Foo>
  */
public function foo() : array
{
    return [new Foo()]
}
```

Reply:
It got fixed by introducing `CommandLineEnvironment` interface which allow to get rid of `Process` in method signatures